test/powershell/Language/Scripting/Debugging/Debugging.Tests.ps1

```diff
@@ -8,32 +9,83 @@ $script1 = @'
 'bb'.ToSTring() > $null
 'bbb'
 '@
+
 $script2 = @'
+$test = 1..2
+switch ($test)
+{
+    default {}
+}
+'@
+
+$script3 = @'
+$test = 2
+for ("string".Length;
```

Comment:
In the product code, there's a special check for while/do loops, but I don't see a test here for that?